test/e2e/workflow_test.go

```diff
@@ -0,0 +1,90 @@
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/argoproj/argo/cmd/argo/commands"
+	. "gopkg.in/check.v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type WorkflowRunSuite struct {
+	clientSet *kubernetes.Clientset
+}
+
+var _ = Suite(&WorkflowRunSuite{})
+
+func (s *WorkflowRunSuite) SetUpSuite(c *C) {
+	// use the current context in kubeconfig
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeConfig)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// create the clientset
+	s.clientSet, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+}
+
+func (s *WorkflowRunSuite) TestRunWorkflowBasic(c *C) {
+	workflowName := "my-test"
+	workflowYaml := `
+apiVersion: argoproj.io/v1alpha1
```

Comment:
This doesn't succeed right now because of https://github.com/argoproj/argo/issues/554.